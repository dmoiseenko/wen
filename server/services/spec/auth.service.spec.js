const authService = require('../auth.service');
const bootstrap = require('../../db/bootstrap/bootstrap.js');


describe('login', () => {
  beforeEach(() => bootstrap());

  it('should provide auth token', async () => {
    const actual = await authService.login({ email: 'js@mail.com', password: 'password' });

    expect(actual).toMatchSnapshot();
  });
});
