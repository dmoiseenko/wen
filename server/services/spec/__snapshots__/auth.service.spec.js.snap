// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login should provide auth token 1`] = `"eyJhbGciOiJIUzI1NiJ9.NWQ3NGQxOTUxYzY1MTVlYw.672rA3xm7fgXNOYpZAtSVO1-NpS3cAwN0Eq3hTxVxjA"`;
